__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const data = require("../db/data/test-data")
+const seed = require("../db/seeds/seed")
+const db = require("../db/connection")
+const app = require("../app")
+const request = require("supertest")
+
+beforeAll(() => seed(data))
+afterAll(() => db.end())
+
+describe("GET/topics", () => {
+    test("200: Should respond with all topics in an array of objects containing the properties slug and description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          const topics = body;
+          topics.forEach((topic) => {
+            expect(typeof topic.slug).toBe("string");
+            expect(typeof topic.description).toBe("string");
+            expect(Object.keys(topic).length).toBe(2)
+          });
```

Comment:
Great assertions to check each topic

I would say though line 20 (checking number of keys) could be quite brittle if in the future the topic data morphed. Arguably we only care that slug and description are there, if there are other keys there it's not as much of a bother